lib/components/Description/DescriptionMeta.tsx

```diff
@@ -3,20 +3,14 @@ import { styles } from './Description.Styles'
 export interface DescriptionMetaProps {
   term?: string
   children: React.ReactNode
-  hasBorder?: boolean
   useColumns?: boolean
-  isSmall?: boolean
 }
 
-export const DescriptionMeta = ({ term, children, hasBorder, useColumns, isSmall }: DescriptionMetaProps) => {
-  const topBorder = hasBorder ? 'first:border-t' : 'first:pt-0'
+export const DescriptionMeta = ({ term, children, useColumns }: DescriptionMetaProps) => {
   const columns = useColumns ? 'md:grid-cols-left-260 gap-2 md:gap-6' : 'grid-row gap-2'
-  const textSize = isSmall
-    ? 'text-base md:text-lg pt-6 pb-6'
-    : 'prose prose-lg md:prose-xl prose-rds max-w-none pt-8 pb-8'
 
   return (
-    <div className={`${styles.base} ${columns} ${topBorder} ${textSize}`}>
+    <div className={`cu-prose ${styles.base} ${styles.baseMeta} ${columns}`}>
```

Comment:
Instead of this, can we add main and section to the story itself?

Reply:
Remove, also removed the not-prose on the parent, don't think it's needed